content/Products/RHOBS/onboarding-a-tenant-into-rhobs.md

```diff
@@ -0,0 +1,84 @@
+# Onboarding a Tenant into Red Hat’s Observability Service
+
+- **Authors:**
+  - [`@squat`](https://github.com/squat)
+
+## Overview
+
+Teams that have identified a need for collecting their logs and/or metrics into a centralized service that offers querying and dashboarding may choose to send their data to Red Hat’s hosted Observatorium instance. This document details the steps required to become a new tenant.
```

Comment:
```suggestion
Teams that have identified a need for collecting their logs and/or metrics into a centralized service that offers querying and dashboarding may choose to send their data to [RHOBS](README.md). This document details the required steps the interested party has to perform to become a new tenant.
```

Reply:
I kind of disagree here, there's no ambiguity IMO. `This document details the steps required to become a new tenant.` Who else would `become a new tenant`? :thinking: